tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -166,8 +166,8 @@ public function testEventManagerPassedToPlatform() : void
     {
         $eventManager = new EventManager();
 
-        /** @var AbstractPlatform|MockObject $driver */
         $platform = $this->createMock(AbstractPlatform::class);
+        assert($platform instanceof AbstractPlatform && $platform instanceof MockObject);
```

Comment:
It feels awkward to make assertions against the values produced by the test framework. As of https://github.com/sebastianbergmann/phpunit/commit/69d23b99e1fa301a28656ea83d68c384cd1a5c4e, it may be no longer needed.

Reply:
I'm going to grep `instanceof MockObject` and remove occurences. It should result in better type information since currently, `||` is used, mostly.